utt/privacy-wallet-service/proto/api/v1/wallet-api.proto

```diff
@@ -98,6 +98,7 @@ message GetStateResponse {
     uint64 budget = 1[jstype = JS_STRING];
     uint64 balance = 2[jstype = JS_STRING];
     string user_id = 3;
+    map<string, uint64> coins = 4;
```

Comment:
how do you attribute jstype = JS_STRING for this map? it is required for JS to work